src/components/App/App.tsx

```diff
@@ -56,17 +59,24 @@ class App extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props)
     this.state = {
-      articles: articles
+      articles: articles,
+      initialSentimentSet: false,
+      userSentiment: 0
     }
   }
+  changeUserSentiment = (newSentiment: number) => {
+  this.setState({userSentiment: newSentiment, initialSentimentSet: true})
+  }
 
   render(): JSX.Element {
     return (
       <div className="App">
-        <header className="App-header">
-          <h1>WellNews</h1>
-        </header>
-        <Feed articles={this.state.articles} />
+        <div className="app-container">
+          <header className="App-header">
+            <h1 className="header-text">WellNews</h1>
+          </header>
+          {this.state.initialSentimentSet ? <Feed articles={this.state.articles} /> : <Form userSentiment={this.changeUserSentiment}/>}
```

Comment:
I would keep the name of the Form's prop of userSentiment consistent with the method name for it to start with a verb in Form as well (i.e., naming it changeUserSentiment as opposed to just userSentiment). What do you think?

Reply:
Can do! I like this suggestion.